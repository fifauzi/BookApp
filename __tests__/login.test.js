import React from 'react';
import {create} from 'react-test-renderer';

import Login from '../src/screens/login/index';
import ContainerTest from '../src/helpers/containerTest';

describe('login snapshot', () => {
  test('should render', () => {
    const component = create(ContainerTest(<Login />)).toJSON();
    expect(component).toMatchSnapshot();
  });
});
