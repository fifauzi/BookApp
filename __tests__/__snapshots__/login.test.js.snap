// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login snapshot should render 1`] = `null`;
